config/bootstrap.php

```diff
@@ -15,7 +15,7 @@
 
 // You can remove this if you are confident that your PHP version is sufficient.
 if (version_compare(PHP_VERSION, '5.5.9') < 0) {
-    trigger_error('You PHP version must be equal or higher than 5.5.9 to use CakePHP.', E_USER_ERROR);
+    trigger_error('Your PHP version must be equal or higher than 5.5.9 to use CakePHP.', E_USER_ERROR);
```

Comment:
That is true, however we should do that as a global possibly. One that gets defined in index.php?